pkg/apis/batch/v1alpha1/job.go

```diff
@@ -52,8 +52,9 @@ type JobSpec struct {
 	SchedulerName string `json:"schedulerName,omitempty" protobuf:"bytes,1,opt,name=schedulerName"`
 
 	// The minimal available pods to run for this Job
+	// Defaults to the summary of tasks' replicas
 	// +optional
-	MinAvailable int32 `json:"minAvailable,omitempty" protobuf:"bytes,2,opt,name=minAvailable"`
+	MinAvailable *int32 `json:"minAvailable,omitempty" protobuf:"bytes,2,opt,name=minAvailable"`
```

Comment:
will that PR break the backward compatability?

Reply:
No, users can use it as before

Reply:
Currently there are some customers already call this Job API in their products.  They have to update their code to adapt this changes.

Reply:
yes, l actually missed that. If user submit the vcjob through code and set/use minAvailable, while not yaml, they need to modify their code indeed.. So what's your opinion about that?

Reply:
We can find another way to resolve the orignal issue and keep the api as it is.

Reply:
The biggest problem with `int` is that we cannot distinguish between user behavior(set to 0) and the default int value of 0. And I think it can only be solved by setting it as a pointer..